src/cli/Peers.ts

```diff
@@ -0,0 +1,387 @@
+import fs from 'fs';
+import commander from 'commander';
+import { PolykeyAgent, PeerInfo } from '../lib/Polykey';
+import {} from '../../tmp/hole-punch-test/publicserver';
```

Comment:
What's this for? If you just want to import something without actually bringing anything. Just do `import 'blah';`, no need for `import {} from 'blah';`.

Reply:
oh, this is just me forgetting to cleanup! I've removed it